### PR TITLE
Fix/various and tests

### PR DESCRIPTION
**Pull request by mrnicegyu11**
- Fixes system test on local machine
- Adds current version of portainer to integrations test
- More integrations tests for portainer
- Adds and tests REST DELETE stack route in portainer
- Various dependency updates
- Updates test's docker-compose file to current images for docker-registry unit test

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -18,6 +18,24 @@
           "remoteRoot": "/devel/services/deployment-agent"
         }
       ]
+    },
```
Comment (pcrespov) [inline .vscode/launch.json:21]: Version ``launch.json.template`` but unversion (ignore) ``launch.json``

Comment (pcrespov) [reply .vscode/launch.json:21]: This is the rule we use in .ignore
```.gitignore

# IDEs config (except templates)
.vscode/*
!.vscode/*.template.json
```

File: tests/unit/test_portainer.py [hunk 2]
```diff
@@ -136,3 +137,29 @@ async def test_create_stack(
...
+    loop: asyncio.AbstractEventLoop,
+    valid_config: Dict[str, Any],
+    portainer_service_mock: aioresponses,
+    aiohttp_client_session: ClientSession,
+    bearer_code: str,
+    portainer_stacks: Dict[str, Any],
+    valid_docker_stack,
+):
+    swarm_id = 1
+    stack_name = "myAmazingstackname"
```
Comment (pcrespov) [inline tests/unit/test_portainer.py:152]: TIP: ``faker`` for fake values  https://faker.readthedocs.io/en/master/

```python
from faker import Faker


test_it(faker: Faker):
   swarm_id = faker.pyint(min=1)
   stack_name = faker.name()
  # ...
```

**Review by pcrespov — CHANGES_REQUESTED** [review]

**Comment by mrnicegyu11** [comment]
THIS PR WAS SPLIT INTO 4 FOR BETTER READABILITY, SEE https://github.com/ITISFoundation/osparc-deployment-agent/pull/154

**Pull request closed without merging**